README.md

```diff
@@ -4,6 +4,15 @@ A project based learning activity for people who are getting started with Git an
 
 You can play the game at: https://githubschool.github.io/github-games/
 
+### Instructions for playing the game
+
```

Comment:
To begin the game

Reply:
Space bar is pressed to begin the game, then to rotate the shape, up and down arrow keys are used. Left and right arrow keys are used to position the shape. The goal of the game is to create complete rows with no empty spaces. Once the rows are completed, they will disappear. To pause, press the space bar again